src/gui/MainWindow.cpp

```diff
@@ -1102,6 +1101,14 @@ void MainWindow::updateViewMenu()
 			toInt() ? true : false );
 	m_viewMenu->addAction(qa);
 	*/
+
+	qa = new QAction(tr( "Enable note labels in piano roll" ), this);
+	qa->setData("printnotelabels");
+	qa->setCheckable( true );
+	qa->setChecked( ConfigManager::inst()->value( "ui", "printnotelabels" ).
+			toInt() ? true : false );
```

Comment:
I wrote that surrounding code, so I guess I just kept on going with a bad habit there. I know I prefer unnecessary explicitness sometimes.